languages/ruby/exercises/concept/hashes/.meta/design.md

```diff
@@ -0,0 +1,43 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept of hashes through [Ruby][ruby-doc-hash].
+
+## Learning objectives
+
+After completing this exercise, the student should:
+
+- Know of the existence of the `hash` collection.
+- Know how to create and initialise a `hash`.
+- Know how to assign values to existing hash keys.
+- Know how to add new hash key.
+- Know how to iterate over a hash.
+
+## Out of scope
+
+- Enumerable methos like select, reject, compact, etc.
+- Memory and performance characteristics.
+
+## Concepts
+
+- `hash`: know of the existence of the `hash` collection; know how to create and initialise a `hash`; know how to assign values to existing hash keys; know how to add new hash key; know how to iterate over a hash.
```

Comment:
Capitalize Hash, because it is a class.  There is also a method of the same name, and we are not covering that here.

Reply:
There is a computer science idea of a hash.  This can be very confusing with this introduction, depending on the students experience.